app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -41,6 +41,8 @@ import org.mozilla.focus.utils.OnUrlEnteredListener
 private const val COL_COUNT = 5
 private const val SETTINGS_ICON_IDLE_ALPHA = 0.4f
 private const val SETTINGS_ICON_ACTIVE_ALPHA = 1.0f
+private const val BUNDLED_TILE_VIEW_TYPE = 0
```

Comment:
nit: Add a line above to separate these constants from the others.

You should group related ideas together so you can quickly scan between groups, rather than having to look at all the individual items in these groups.